testdata/local/tasks.json

```diff
@@ -8,14 +8,12 @@
-{"_id":"evergreen_ubuntu1604_test_annotations_b_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48","secret":"211937152e30222da24aac8c8779af81","create_time":{"$date":"2020-02-20T20:37:06Z"},"injest_time":{"$date":"2020-02-21T15:13:48.838Z"},"dispatch_time":{"$date":"2020-02-21T15:14:32.455Z"},"scheduled_time":{"$date":"2020-02-21T15:14:02.452Z"},"start_time":{"$date":"2020-02-21T15:14:35.244Z"},"finish_time":{"$date":"2020-02-21T15:29:54.786Z"},"activated_time":{"$date":"2020-02-21T15:13:48.838Z"},"version":"5e4ff3abe3c3317e352062e4","branch":"evergreen","gitspec":"5e823e1f28baeaa22ae00823d83e03082cd148ab","priority":{"$numberLong":"0"},"task_group":"","last_heartbeat":{"$date":"2020-02-21T15:29:35.243Z"},"activated":true,"activated_by":"","build_id":"evergreen_ubuntu1604_patch_5e823e1f28baeaa22ae00823d83e03082cd148ab_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48","distro":"ubuntu1604-build","build_variant":"ubuntu1604","build_variant_display_name":"Ubuntu 16.04","depends_on":[],"distro_aliases":["ubuntu1604-build"],"display_name":"test-cloud","tags":["db","test"],"host_id":"i-0befc05877e1dba0c","restarts":0,"execution":0,"order":2567,"r":"github_pull_request","status":"failed","time_taken":{"$numberLong":"919542945866"},"duration_prediction":{"value":{"$numberLong":"414932876125"},"std_dev":{"$numberLong":"267962094193"},"ttl":{"$numberLong":"30843773435822"},"collected_at":{"$date":"2020-02-21T15:14:02.245Z"},"expected_duration":{"$numberLong":"414932876125"},"expected_duration_std_dev":{"$numberLong":"267962094193"}},"details":{"status":"failed","type":"setup","desc":"'host.list' in \"setup-docker-host\" (#2)","timed_out":true,"timeout_type":"exec"},"logs":{"agent":null,"system":null,"task":null},"display_task_id":""}
```

Comment:
this `test-cloud` was a Known Issue so if it has any task annotations they should probably be deleted from local data too... or you could just delete the other `test-cloud`

Reply:
Wow great catch!